tests/func/tenant/seed/router.yml

```diff
@@ -11,3 +11,20 @@
       - net: osm_net
         subnet: osm_router_subnet
         portip: 192.168.10.10
+  register: _osm_router
+
+- name: get auth token
+  os_auth:
+    auth: "{{ os_migrate_src_auth }}"
+
+- name: set tags on router
+  shell: |
+    export OS_AUTH_TYPE=token
+    export OS_AUTH_URL="{{ os_migrate_src_auth.auth_url }}"
+    export OS_TOKEN="{{ auth_token }}"
+    export OS_PROJECT_ID="{{ os_migrate_src_auth.project_id|default('') }}"
+    export OS_PROJECT_NAME="{{ os_migrate_src_auth.project_name|default('') }}"
+    export OS_PROJECT_DOMAIN_ID="{{ os_migrate_src_auth.project_domain_id|default('') }}"
+    export OS_PROJECT_DOMAIN_NAME="{{ os_migrate_src_auth.project_domain_name|default('') }}"
```

Comment:
Are these exported using shell for debug value? If not, I'd suggest using the 'environment' argument on the task instead. Also, this could use command instead of shell.

Reply:
Thanks, i'll try that.